script/Ciel/Day 7/07_01_CIEL07_2_2.txt

```diff
@@ -95,6 +95,7 @@ My ears are being bombarded by whispers.
 -- Page 290, Offset 15490.
 -- こういう奇襲を受けるのならいくらでも牧場に住もう、と思わなくもなくもない。
 // 'all the time' is an addition that might not be necessary
+// Is this a metaphor for being attacked by a wolf or something?
```

Comment:
Noel calls him a lamb > noel is 'attacking' him > shiki likes it > maybe 'being a lamb = living in the pastures' isn't that bad?